rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.81"
+channel = "1.83"
```

Comment:
I am really not a fan of this, as - especially for risc0-ethereum - it leads to super cryptic error messages, as alloy has already switched to rust 1.83, but all these crates are not (yet) supported in our ZKVM.

Reply:
Has been addressed in ab9731892614ea0a4afc82ad82f878c224a13255